### ISSUE TITLE
Mine of Phandelver DNDbeyond sourcebook descriptive blocks not longer able to "display in vtt"

### ISSUE THREAD

**Issue by radicalrude** [kakaroto/Beyond20#996]
**Describe the bug**
Mine of Phandelver DNDbeyond sourcebook descriptive blocks not longer able to "display in vtt"

**To Reproduce**
on any computer with dndbeyond sourcebook mine of phandelver sourcebook bought search for a descriptive text block, and it is not able to display in vvt

**Expected behavior**
red button should be there, any other sourcebook campaign still works

**Screenshots**
![image](https://user-images.githubusercontent.com/108473604/176693510-54bee747-9054-43be-b995-8bbc1d8ced8b.png)

**Browser Info (please complete the following information):**
chrome OS

**Additional context**
any other sourcebook works, seems layout mine of phandelver is changed?

**Comment by Aeristoka** [comment]
@radicalrude got links to where this isn't working so I can check it quick?

(I would bet DnDBeyond did another formatting change...)

**Comment by radicalrude** [comment]
> @radicalrude got links to where this isn't working so I can check it quick?
> 
> (I would bet DnDBeyond did another formatting change...)

https://www.dndbeyond.com/sources/lmop/the-spiders-web#TriboarTrail

looks like it yeah

**Comment by radicalrude** [comment]
the whole sourcebooks looks different

**Comment by Aeristoka** [comment]
@radicalrude I think I've got it fixed with https://github.com/kakaroto/Beyond20/pull/997

Waiting on @kakaroto to review and see if I missed something obvious.

**Comment by radicalrude** [comment]
@Aeristoka just checking, this will be fixed in the next release?

**Comment by Aeristoka** [comment]
Sure will! Not 100% sure when that will be, but it has been merged into the Beyond20 code.

In the meantime, if you want the fix right this instant, you can install a Dev version of Beyond20: https://github.com/kakaroto/Beyond20#developer-mode-installation

Just be aware that the dev version doesn't auto update, and you'd have to remove it and re-enable the official version when there was a new release.
